packages/netlify-cms-core/src/components/Collection/Entries/Entries.js

```diff
@@ -13,34 +24,40 @@ const Entries = ({
   cursor,
   handleCursorActions,
   t,
+  page,
 }) => {
   const loadingMessages = [
     t('collection.entries.loadingEntries'),
     t('collection.entries.cachingEntries'),
     t('collection.entries.longerLoading'),
   ];
 
-  if (entries) {
```

Comment:
`entries` is always defined (couldn't find a flow it wasn't) so the loader was never shown